ly/musicxml/xml_objs.py

```diff
@@ -336,7 +340,6 @@ def merge_lyrics(self, lyrics):
                             obj.add_lyric(l)
                         i += 1
 
-
```

Comment:
Sorry to nitpick and this is probably a simple oversight from your part! But it should be two blank lines before the class definition. https://www.python.org/dev/peps/pep-0008/#blank-lines